sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -192,6 +192,17 @@ case class InsertIntoHiveTable(
     if (partition.nonEmpty) {
       if (numDynamicPartitions > 0) {
         if (overwrite && table.tableType == CatalogTableType.EXTERNAL) {
+          val numWrittenParts = writtenParts.size
+          val maxDynamicPartitionsKey = "hive.exec.max.dynamic.partitions"
```

Comment:
can we use `HiveConf.ConfVars.DYNAMICPARTITIONMAXPARTS.varname` for the key and `HiveConf.ConfVars.DYNAMICPARTITIONMAXPARTS.defaultIntVal` instead of `1000`?